build-logic/src/main/kotlin/dokkabuild.base.gradle.kts

```diff
@@ -23,7 +23,7 @@ tasks.withType<AbstractArchiveTask>().configureEach {
     isReproducibleFileOrder = true
 }
 
-val integrationTestPreparation by tasks.registering {
+tasks.register("integrationTestPreparation") {
```

Comment:
Can you explain more about this change? Personally I prefer the delegated accessor version over the string-based version.

Reply:
It's just shorter and simpler. If you prefer the other way I'll revert.